src/options.js

```diff
@@ -11,8 +13,16 @@ var defaults = {
   bracketSpacing: true
 };
 
+var exampleConfig = Object.assign({}, defaults, {
+  filename: "testFilename",
+  useFlowParser: false,
+  printWidth: 80,
+  originalText: "text"
+});
+
 // Copy options and fill in default values.
 exports.normalize = function(options) {
+  validate(options, { exampleConfig });
```

Comment:
Also we can move the call to `normalize` up to `index.js` so we can do it before we attach `originalText` and other internal options.